examples/vpc-cni-custom-networking/README.md

```diff
@@ -0,0 +1,132 @@
+  Allocatable:
+    attachable-volumes-aws-ebs:  25
+    cpu:                         3920m
+    ephemeral-storage:           95551679124
+    hugepages-1Gi:               0
+    hugepages-2Mi:               0
+    memory:                      14902292Ki
+    pods:                        110 # <- this should be 110 and not 58
+```
+
+4. List out the pods running currently:
+
+```sh
+kubectl get pods -A -o wide
+
+# Output should look similar to below
+NAMESPACE     NAME                       READY   STATUS    RESTARTS   AGE   IP            NODE                                       NOMINATED NODE   READINESS GATES
+kube-system   aws-node-ttg4h             1/1     Running   0          52s   10.0.34.74    ip-10-0-34-74.us-west-2.compute.internal   <none>           <none>
+kube-system   coredns-657694c6f4-8s5k6   1/1     Running   0          2m    10.99.135.1   ip-10-0-34-74.us-west-2.compute.internal   <none>           <none>
+kube-system   coredns-657694c6f4-ntzcp   1/1     Running   0          2m    10.99.135.0   ip-10-0-34-74.us-west-2.compute.internal   <none>           <none>
+kube-system   kube-proxy-wnzjd           1/1     Running   0          53s   10.0.34.74    ip-10-0-34-74.us-west-2.compute.internal   <none>           <none>
+```
+
+5. Inspect one of the `aws-node-*` (AWS VPC CNI) pods to ensure prefix delegation is enabled and warm prefix target is 1:
+
+```sh
+kubectl describe pod aws-node-ttg4h -n kube-system
+
+# Output should look similar below (truncated for brevity)
+  Environment:
+    ADDITIONAL_ENI_TAGS:                    {}
+    AWS_VPC_CNI_NODE_PORT_SUPPORT:          true
+    AWS_VPC_ENI_MTU:                        9001
+    AWS_VPC_K8S_CNI_CONFIGURE_RPFILTER:     false
+    AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG:     true # <- this should be set to true
+    AWS_VPC_K8S_CNI_EXTERNALSNAT:           false
+    AWS_VPC_K8S_CNI_LOGLEVEL:               DEBUG
+    AWS_VPC_K8S_CNI_LOG_FILE:               /host/var/log/aws-routed-eni/ipamd.log
+    AWS_VPC_K8S_CNI_RANDOMIZESNAT:          prng
+    AWS_VPC_K8S_CNI_VETHPREFIX:             eni
+    AWS_VPC_K8S_PLUGIN_LOG_FILE:            /var/log/aws-routed-eni/plugin.log
+    AWS_VPC_K8S_PLUGIN_LOG_LEVEL:           DEBUG
+    DISABLE_INTROSPECTION:                  false
+    DISABLE_METRICS:                        false
+    DISABLE_NETWORK_RESOURCE_PROVISIONING:  false
+    ENABLE_IPv4:                            true
+    ENABLE_IPv6:                            false
+    ENABLE_POD_ENI:                         false
+    ENABLE_PREFIX_DELEGATION:               true # <- this should be set to true
+    MY_NODE_NAME:                            (v1:spec.nodeName)
+    WARM_ENI_TARGET:                        1 # <- this should be set to 1
+    WARM_PREFIX_TARGET:                     1
+    ...
+```
+
+## Destroy
+
+To teardown and remove the resources created in this example:
+
+```sh
+terraform destroy -auto-approve
```

Comment:
Single TF destroy works without any known ENI dangling issues?

Reply:
yes, but let me update to align with what we've been recommending